discovery/consul/consul.go

```diff
@@ -184,22 +182,16 @@ func NewDiscovery(conf *SDConfig, logger log.Logger) (*Discovery, error) {
 		logger = log.NewNopLogger()
 	}
 
-	tls, err := config.NewTLSConfig(&conf.TLSConfig)
+	httpConfig := config.HTTPClientConfig{
+		TLSConfig: conf.TLSConfig,
```

Comment:
As a temporary workaround, can you add `FollowRedirects: true,`, so we do not break users?

Reply:
Yes, I wondered about that, misread the `client.CheckRedirect` configuration – thanks for the catch.